### ISSUE TITLE
Include ogr_fdw extension

### ISSUE THREAD

**Issue by robe2** [postgis/docker-postgis#209]
As noted in this ticket - https://github.com/pramsey/pgsql-ogr-fdw/issues/182

It would be really handy if docker-postgis files include ogr_fdw.

ogr_fdw spatial data wrapper is already carried by apt.postgresql.org and yum.postgresql.org and its big dependency is GDAL which is already included with the postgis extensions.

I can take a stab at adding it if all are okay with the idea.

**Comment by phillipross** [comment]
@robe2 there's currently an open item to actually go the opposite way and remove extensions that have traditionally been included with the goal of providing a lighter weight image to serve as a base that people can extend.  (see #187)

There are already some workable ideas on the table about how to accomplish this while avoiding breaking backward compatible, maintaining compatibility with the upstream postgres docker image, etc.  There are still some parts of the problem that need to be addressed though, such as how to deal with alpine, which kinds of builds to provide (stable, edge, master), and then how to provide a good way to make the images upgradable from version to version.  Please feel free to join the discussion on that issue.

**Comment by robe2** [comment]
Just a note here.  I propose providing the extension libs for ogr-fdw, NOT to install the extension.  I'll put my other comments on the #187 ticket.

**Comment by phillipross** [comment]
Yes, definitely... we'll keep this issue open here as the proposal for adding ogr-fdw to the list of optional extensions to be installed on deployment of the container.  The mechanisms to get those extensions deployable and upgradable are what #187 is about.

**Comment by robe2** [comment]
@phillipross  and @ImreSamu is the main issue with including ogr_fdw extension the building of it. I realize now that since most of your PostGIS related things are built rather than installed from packages, it probably makes more sense to compile ogr_fdw as part of the docker image rather than installing from packages.  It will add insignificant weight to the install, since it's just a thin wrapper around GDAL vector support that you already have installed in the image.

Note I'm not talking about having it pre-installed as one of the pre-installed extensions, just having it available if a user wants to install it in a PostGIS enabled database.  If that is the main stumbling block, I'm happy to take a stab at providing a pull request for it.
